src/baseline.rs

```diff
@@ -192,6 +192,51 @@ impl RegistryBaseline {
     pub fn set_version(&mut self, version: semver::Version) {
         self.version = Some(version);
     }
+
+    // To get the rustdoc of the baseline, we first somewhere create a dummy project
+    // with the baseline as a dependency and on in we launch `cargo rustdoc`.
```

Comment:
Consider making this a doc-comment `///` instead of just a regular comment? Suggested slight rephrasing for clarity and minor typo fixes:
```suggestion
    /// To get the rustdoc of the baseline, we first create a placeholder project somewhere
    /// with the baseline as a dependency, and run `cargo rustdoc` on it.
```